mmv1/templates/terraform/examples/security_policy_rule_default_rule.tf.erb

```diff
@@ -0,0 +1,37 @@
+resource "google_compute_security_policy" "default" {
+  name        = "<%= ctx[:vars]['sec_policy_name'] %>"
+  description = "basic global security policy"
+  type        = "CLOUD_ARMOR"
+}
+
+# A default rule is generated when creating the security_policy resource, import is needed to patch it
```

Comment:
The same issue exists in the regional security policy rule resource: https://github.com/hashicorp/terraform-provider-google/issues/15687